soc/arm/nxp_imx/rt/soc_rt10xx.c

```diff
@@ -216,15 +194,6 @@ static ALWAYS_INLINE void clock_init(void)
 	CLOCK_SetMux(kCLOCK_CanMux, 2); /* Set Can clock source. */
 #endif
 
-#if !(defined(CONFIG_CODE_FLEXSPI) || defined(CONFIG_CODE_FLEXSPI2)) && \
```

Comment:
Is this part tested on any boards?

Reply:
@hakehuang ran the full test suite on the boards. 
We are running in XIP mode, hence we should not be touching the FlexSPI clock. The PLL has been setup by ROM using the DCD information.